lib/chef/client.rb

```diff
@@ -756,50 +705,12 @@ def converge_and_save(run_context)
         begin
           save_updated_node
         rescue Exception => e
-          raise e if Chef::Config[:audit_mode] == :disabled
-          converge_exception = e
+          raise e
```

Comment:
you probably still need `converge_exception = e`

Reply:
okay i see this one as well now, "converge_exception = e" should not be hit if we raise.

Reply:
```
def converge_and_save(run_context)
  converge_exception = converge(run_context)
  unless converge_exception
    save_updated_node
  end
  converge_exception
end
```

i don't particularly like the way that the node save looks like it clobbers the shit out of the converge exception, but i think that is backwards compatible with what we've always done.